sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -1545,28 +1545,6 @@
       </build>
     </profile>
 
-    <profile>
-      <id>external-dependency-version-overrides</id>
```

Comment:
Should we keep this profile to use it for the next round of RC tests? We could probably skip this profile 
```xml
<configuration>
  <skip>true</skip>
</configuration>
```

Reply:
Let's split the difference, I'll comment it out and add an explanation about it